microraiden/microraiden/client/client.py

```diff
@@ -171,15 +175,16 @@ def open_channel(self, receiver_address: str, deposit: int):
             self.context.channel_manager,
             'ChannelCreated',
             from_block=current_block + 1,
+            to_block='latest',
             argument_filters=filters
         )
 
         if event:
             log.debug('Event received. Channel created in block {}.'.format(event['blockNumber']))
             channel = Channel(
                 self.context,
-                event['args']['_sender_address'],
-                event['args']['_receiver_address'],
+                self.context.address,
```

Comment:
Maybe add assert to test if event addr is equal to stored one.

Reply:
Added.